Depth_First_Search/Depth_First_Search.dart

```diff
@@ -0,0 +1,74 @@
+// Required library to use inbuilt queue in LIFO manner in Dart
+import 'dart:collection';
+// Required library to use standard output facility of dart
+import 'dart:io';
+
+// Function to find the depth first search of a graph
+void Depth_First_Search(var graph,int source,int no_vertices){
+    // Queue is used in LIFO manner to obtain stack functionality.
+    var queue=Queue();
+    // A list to keep track of visited nodes
+    var visited=new List(no_vertices);
+    // Initializing the list
+    for(int i=0;i<no_vertices;i++){
+        visited[i]=0;
+    }
+    // Adding the source node to LIFO queue ( stack )
+    queue.add(source);
+    // Marking the node visited
+    visited[source]=1;
+    int val;
+
+    while(queue.isNotEmpty){
+        val=queue.removeLast();
+        stdout.write("$val ");
+        
+        //Visiting neighbours of the popped node  
+        for(int i=0;i<no_vertices;i++){
+            if(graph[val][i]==1&&visited[i]==0){
+                queue.add(i);
+                visited[i]=1;
+            }
+        }
+    } 
+}
+
+void main(){
+
+    int no_vertices=5;
```

Comment:
Please take dynamic inputs wherever applicable

Reply:
Made changes to take input dynamically